arcade/paths.py

```diff
@@ -81,9 +124,22 @@ def get_vertex_neighbours(self, pos: Point) -> List[Tuple[float, float]]:
             n.append((x2, y2))
         return n
 
-    def move_cost(self, a: Point, b: Point):
+    def move_cost(self, a: Point, b: Point) -> float:
+        """
+        Returns a float of the cost to move
+
+        moving to corners costs more than to the side.
```

Comment:
```suggestion
        Moving diagonally costs more than along the X & Y axes.
```
1. Capitalization
2. It's not just side to side, but vertically as well.